starky/src/prover.rs

```diff
@@ -164,34 +162,44 @@ where
 {
     let degree = 1 << degree_bits;
 
+    let max_degree_bits = log2_ceil(stark.constraint_degree() - 1);
```

Comment:
I know this matches the PLONK code but the name seems kinda unclear to me; maybe something like `evaluation_domain_bits` would be more fitting

Reply:
Hmm I agree `max_degree_bits` isn't a very good name, but I would interpret `evaluation_domain_bits` as the log of the LDE size. What do you think about `constraint_degree_bits` or `quotient_degree_bits`?

Reply:
`quotient_degree_bits` sounds good to me